src/cljs/main/broadfcui/utils.cljs

```diff
@@ -116,16 +116,23 @@
 ;; TODO - make this unnecessary
 (def content-type=json {"Content-Type" "application/json"})
 
-(defn ajax [arg-map]
-  (let [url (:url arg-map)
-        on-done (:on-done arg-map)
-        method (if-let [method (:method arg-map)] (string/upper-case (name method)) "GET")
-        headers (:headers arg-map)
-        data (:data arg-map)
-        with-credentials? (:with-credentials? arg-map)
-        canned-response-params (when-not @use-live-data? (:canned-response arg-map))]
+(defonce ^:private recent-ajax-calls (atom #{}))
+(def ^:private double-call-threshold 500)
```

Comment:
I suggest increasing this threshold to at least 2 seconds, maybe more. 500ms wouldn't catch, for instance, a (serial) double request to the workspace list, since a single request is > 500ms.